docs/content/interacting/managing-relationships-between-objects.mdx

```diff
@@ -132,6 +143,12 @@ Another way of modeling this is to have an authorization model as follows:
             this: {},
           },
         },
+        metadata: {
+          relations: {
+            owner: { directly_related_user_types: [{type: 'org'}] },
+            admin: { directly_related_user_types: [{type: 'user'},{type: 'org', relation: 'repo_admin'}] },
```

Comment:
Why {type: 'org', relation: 'repo_admin'}? We don't do any writes on that, that is just a tuple to userset

Reply:
```suggestion
            admin: { directly_related_user_types: [{type: 'user'}] },
```